### PR TITLE
Configure logrotate when systemd is not used

### PR DESCRIPTION
**Pull request by elemoine**

### CODE REVIEW

**Review by simonpasquier — APPROVED** [review]

**Comment by elemoine** [comment]
I removed the `su` option as suggested by @SwannCroiset in https://github.com/tcpcloud/salt-formula-collectd/pull/39/files.

File: heka/files/heka_logrotate.conf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  delaycompress
+  missingok
+  notifempty
+  # logrotate allows to use only year, month, day and unix epoch
+  dateext
+  dateformat -%Y%m%d-%s
+  # number of rotated files to keep
+  rotate 10
+  # do not rotate files unless both size and time conditions are met
+  hourly
```
Comment (SwannCroiset) [inline heka/files/heka_logrotate.conf:15]: 'hourly' is useless b/c logrotate is executed daily basis.
If we absolutely need  to rotate hourly we must run a dedicated crontab for that (and don't put this conf in standard logrotate conf directory).
see https://github.com/openstack/fuel-plugin-lma-collector/blob/bb67a130628477de4e3d23e9490ed854e7fd959f/deployment_scripts/puppet/modules/heka/manifests/init.pp#L224-L252

and 
https://github.com/openstack/fuel-plugin-lma-collector/blob/bb67a130628477de4e3d23e9490ed854e7fd959f/deployment_scripts/puppet/modules/heka/templates/logrotate.cron.erb#L33

**Review by SwannCroiset — CHANGES_REQUESTED** [review]

File: heka/files/heka_logrotate.conf (hunk 1 above)
Comment (elemoine) [reply heka/files/heka_logrotate.conf:15]: I guess we need to do the same as the Fuel plugin, or the same problems will eventually resurface. Thanks a lot for catching this.

**Comment by elemoine** [comment]
@SwannCroiset I added a commit that introduces the specific logrotate script and crontab, as done by the Fuel plugin.

**Review by SwannCroiset — APPROVED** [review]

**Pull request merged**